GoogleCloudPlatform/kubernetes-engine-samples
# Configure bot: License Header Lint GCF

* See https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/issues/963.
* I have enabled [License Header Lint GCF](https://github.com/apps/license-header-lint-gcf) in this repo.
* This pull-request configures the bot.
* The bot simply runs a GitHub check called `header-check` that warns us about invalid/missing license headers.
* How to review:
    * ensure configuration file looks fine (e.g., list of file extensions)
    * ensure that the bot is actually working inside this pull-request
        * You can see that this bot works in the [commits from this pull-request](https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/pull/964/commits).
        * Look at the commits titled:
            * [Temporarily use incorrect license year](https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/pull/964/commits/b71ff8c0da85440a2dab2f5ecedb24ad03b18b37)
            * [Temporarily remove license header](https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/pull/964/commits/035a79485bd1bcbdd11d09aea02bb61addea85a0)
* This bot does not add the headers for us. It merely warns us of missing/invalid headers.
